### PR TITLE
Update README-zh.md

### PR DESCRIPTION
**Pull request by river42069**

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Usage:weex <command>
 
 where <command> is one of:
 
-       init                                   创建一个weex项目
+       init                                   创建一个weex项目 (已删除)
        debug                                  调试
        compile                                打包weex文件
        create                                 创建一个weexpack工程
```
Comment (erha19) [inline README-zh.md]: This should be `创建一个weex工程`

File: README-zh.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Usage:weex <command>
 
 where <command> is one of:
 
-       init                                   创建一个weex项目
+       init                                   创建一个weex项目 (已删除)
```
Comment (erha19) [inline README-zh.md]: Change `删除` to `弃用`

**Review by erha19 — COMMENTED** [review]
Fix it and  I will merge this pr.

**Review by river42069 — COMMENTED** [review]
Update README-zh.md

**Pull request merged**